core-services/image-mirroring/openshift/mapping_origin_4_10

```diff
@@ -113,6 +113,7 @@ registry.ci.openshift.org/origin/4.10:external-dns-operator quay.io/openshift/or
 registry.ci.openshift.org/origin/4.10:f5-router quay.io/openshift/origin-f5-router:4.10 quay.io/openshift/origin-f5-router:4.10.0 quay.io/openshift/origin-f5-router:latest
 registry.ci.openshift.org/origin/4.10:federation-controller quay.io/openshift/origin-federation-controller:4.10 quay.io/openshift/origin-federation-controller:4.10.0 quay.io/openshift/origin-federation-controller:latest
 registry.ci.openshift.org/origin/4.10:file-integrity-operator quay.io/openshift/origin-file-integrity-operator:4.10 quay.io/openshift/origin-file-integrity-operator:4.10.0 quay.io/openshift/origin-file-integrity-operator:latest
+registry.ci.openshift.org/origin/4.10:frr quay.io/openshift/origin-frr:4.10 quay.io/openshift/origin-frr:4.10.0 quay.io/openshift/origin-frr:latest
```

Comment:
if this is part of metal LB, and has no function outside of it, then it would be prefixed under that.  something like `metallb-frr`.  I assume that this image would be used to serve routing protocol information on various nodes?

Reply:
> if this is part of metal LB, and has no function outside of it, then it would be prefixed under that. something like `metallb-frr`. I assume that this image would be used to serve routing protocol information on various nodes?

correct

Reply:
Ok, then I'll approve `metallb-frr` once you make that change.